macro/Version.py

```diff
@@ -47,10 +47,10 @@
 def execute(macro, args):
     if args:
         version = str(args)
-        if version.lower() == 'jade':
+        if version.lower() == 'kame':
```

Comment:
Following the tradition of using the first word for codenames, shouldn't this be kinetic?